SU-SWS/react_paragraphs
# Provide error handeling for media library widget

# READY FOR REVIEW

# Summary
- Provide a message if the media entity is no longer around

# Review By (Date)
- 4/22

# Urgency
- High

# Steps to Test
1. checkout this branch
1. clear browser cache
1. create a page with a card and add an image & save the page
1. navigate to `/admin/content/media` and delete the media you used
1. go back to edit the page you made.
1. validate the form doesn't blow up when you open the paragraph modal. It should give you a message that no preview is available.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
